File: Q12/main.c
#include<stdio.h>
int main(){
    int n;
    scanf("%d",&n);
    int arr[n];
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    int start=0,target=1;

    while(start<n){
        if(arr[start]==target){
            start++;target++;
        }
        else{
            printf("%d",target);
            return 0;
        }
    }
    printf("%d",target);
    return 0;
}

File: Q12/main.py
n=int(input())
arr=[]
for i in range(n):
    arr.append(int(input()))

start,target=0,1

while start<n:
    if arr[start]==target:
        start+=1
        target+=1
    else:
        print(target)
        exit()
print(target)